lib/lib/ship.rb

```diff
@@ -0,0 +1,34 @@
+  class Ship
+
+  attr_accessor :name, :length, :health
```

Comment:
Because I didn't quite get the difference. Thanks for challenging me to research and understand better. I get that it's a reader now and "protects" the code from being set :)

Reply:
Note to Self - [Article](https://mixandgo.com/learn/ruby_attr_accessor_attr_reader_attr_writer) on accessor v reader v writer